src/adagrams.js

```diff
@@ -1,15 +1,74 @@
+const letterPool = {
+  A: 9, B: 2, C: 2, D: 4, E: 12, F: 2, G: 3, H: 2, I: 9, J: 1, K: 1, L: 4, M: 2,
+  N: 6, O: 8, P: 2, Q: 1, R: 6, S: 4, T: 6, U: 4, V: 2, W: 2, X: 1, Y: 2, Z: 1
+};
+
 export const drawLetters = () => {
-  // Implement this method for wave 1
+  const letterPoolCopy = JSON.parse(JSON.stringify(letterPool));
```

Comment:
This works well to create a (mostly) deep copy, but consider whether you need a deep copy here, since there's no nested data, you can get away with a shallow copy, which is a little easier to read:
```js
const letterPoolCopy = { ...letterPool };
```

Reply:
That does read so much better!